frontend/package.json

```diff
@@ -33,12 +33,13 @@
 		"@mdx-js/loader": "2.3.0",
 		"@mdx-js/react": "2.3.0",
 		"@monaco-editor/react": "^4.3.1",
+		"@types/axios": "0.14.0",
```

Comment:
<img width="1221" alt="Screenshot 2023-12-10 at 12 57 52 PM" src="https://github.com/SigNoz/signoz/assets/88981777/f533f5f9-2d2d-4d6b-bdb4-54037507886b">

Do you think this dependency is needed? (refer image above)

Reply:
it's for types sync.

Reply:
No No, this isn't required. Had installed while was facing some type errors.